modules/applications/pages/download-and-launch.adoc

```diff
@@ -6,6 +6,11 @@
 For Subscription editions only.
 ====
 
+== Pre-requisites 
+* Docker installed
+* Nginx installed
```

Comment:
Isn't nginx setup using a Docker image ?

Reply:
I'm not sure. I had to `brew install nginx` and manually create a nginx.conf file